src/js/xAxisTimeSeries.js

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Mix-in grade for time-series x axis
+
+    fluid.defaults("floe.chartAuthoring.xAxisTimeSeries", {
+        model: {
+            dataSet: []
+        },
+        axisOptions: {
+            numberOfXAxisTicks: 6
```

Comment:
Maybe it would make sense to push this & the `drawYAxis` functionality into the general `axis` grade? Alternately, separate `xAxis` and `yAxis` grades?